### DISCUSSION TITLE
APOE: error

### DISCUSSION THREAD

**Discussion by krootesmurdy** [trendscenter/coinstac#1548] [General]
Running on two machines as two sites, was working just fine before. Only change was moving wifis.

"Pipeline error from pipeline 6348195f16c71f03e531d58b user: test3\n Error details: Computation failed with exitcode 1 and stderr Traceback (most recent call last):\n File \"/computation/scripts/local.py\", line 187, in <module>\n sys.stdout.write(local_0(PARAM_DICT))\n File \"/computation/scripts/local.py\", line 36, in local_0\n covar_x = average_nifti(args)\n File \"/computation/scripts/nipype_utils.py\", line 66, in average_nifti\n image_data = nib.load(os.path.join(input_dir, image)).dataobj[:]\n File \"/usr/local/lib/python3.7/site-packages/nibabel/loadsave.py\", line 100, in load\n img = image_klass.from_filename(filename, **kwargs)\n File \"/usr/local/lib/python3.7/site-packages/nibabel/dataobj_images.py\", line 472, in from_filename\n keep_file_open=keep_file_open)\n File \"/usr/local/lib/python3.7/site-packages/nibabel/analyze.py\", line 968, in from_file_map\n with hdr_fh.get_prepare_fileobj(mode='rb') as hdrf:\n File \"/usr/local/lib/python3.7/site-packages/nibabel/fileholders.py\", line 70, in get_prepare_fileobj\n obj = ImageOpener(self.filename, *args, **kwargs)\n File \"/usr/local/lib/python3.7/site-packages/nibabel/openers.py\", line 150, in __init__\n self.fobj = opener(fileish, *args, **kwargs)\nFileNotFoundError: [Errno 2] No such file or directory: '/input/6346ef18748eab03d2fd1a14/6348195f16c71f03e531d58b/10048-swc1Re.nii'\n"

**Comment by krootesmurdy** [comment]
I've re-mapped both sites and re-started COINSTAC, still getting the same error.

**Reply by krootesmurdy** [reply]
Re-starting Docker on both machines removed the error, regression running fine now.

**Comment by rssk** [comment]
issue w yann
```Oct 13 21:51:53 ip-10-120-1-240 node[7355]: /usr/local/src/coinstac/packages/coinstac-pipeline/src/express-file-server-setup.js:27
Oct 13 21:51:53 ip-10-120-1-240 node[7355]:       const fp = path.join(activePipelines[req.body.runId].baseDirectory, req.body.clientId);
Oct 13 21:51:53 ip-10-120-1-240 node[7355]:                                                            ^
```
